docs/docs/how-to/performance/improving-site-performance.md

```diff
@@ -0,0 +1,203 @@
+Gatsby inlines CSS into the initial HTML file. So when running Coverage look for the line with the current page's UR and take note of the % of bytes that are unused and the page size.
+
+A "good" scenario is having between 20-40% of unused code -- this includes, for example CSS to define responsive layouts that isn't evaluated on desktop format. A "bad" scenario is 80-90% unused code and a 1mb file.
+
+You can dig deeper by scroll through the page on the usage drawer to look at the unused CSS and get a sense of whether it is needed for that page. To fix these issues, look at moving to a modular CSS solution [like CSS modules](/docs/how-to/styling/css-modules/).
+
+#### Second, if you're using a CSS-in-JS library, use the Gatsby plugin
+
+If you're using a CSS-in-JS library like styled-components or emotion, use the relevant plugin: [gatsby-plugin-styled-components](https://www.gatsbyjs.com/plugins/gatsby-plugin-styled-components/) or [gatsby-plugin-emotion](https://www.gatsbyjs.com/plugins/gatsby-plugin-emotion/).
+
+These plugins server-side render the styles; otherwise, the output HTML will intersperse `<style>` tags with HTML elements, which can cause the browser to perform costly layout reflow.
+
+#### Third, optimize fonts
+
+Font files can usually be reduced in size significantly. If your font file is over 50kb, it's too large. In addition, fonts block page load, so it's important to think about reducing network calls.
+
+- **Prefer `woff2`. Don't use `ttf`.** `woff2` is a compressed font format, supported by [browsers used by over 95% of Internet users](https://caniuse.com/woff2). [A few legacy browsers need `woff`](https://caniuse.com/woff).Like using `avif` and `webp` instead of `png` and jpg, using the correct format can significantly cut down the amount of data sent over the network.  
+- **Self-host rather than installing from an external CDN.** Having the font file available locally will save a trip over the network and reduce blocking time. 
+- **Use Latin font subsets only** (if creating a Latin-language site). It's common to accidentally include font extensions (Greek, Cyrillic, Devnagari, Chinese) when typically you only need the Latin base set. The [Google Webfonts Helper app](https://google-webfonts-helper.herokuapp.com/fonts/SourceSansPro) can help you do this with free fonts.
+
+Font optimizations are usually small, but easy performance wins. 
+
+### Images & Media
+
+Media files are often the largest files on a site, and so can delay page load significantly while they are pulled over the network, especially if their location is not well-defined.
+
+[Gatsby Image](/docs/how-to/images-and-media/using-gatsby-image/) is our approach to optimizing image loading performance. It does three basic things:
+
+- First, it delays non-essential work for images not above the fold to avoid esource congestion. 
+- Second, it provides a placeholder during image fetch.
+- Third, it minimizes image file size to reduce request roundtrip time.
```

Comment:
```suggestion
1. It delays non-essential work for images not above the fold to avoid esource congestion. 
2. It provides a placeholder during image fetch.
3. It minimizes image file size to reduce request roundtrip time.
```